src/Sentry/Sentry.csproj

```diff
@@ -58,7 +58,7 @@
   </ItemGroup>
 
   <!-- Prefer bundled version of System.Text.Json to avoid extra dependencies -->
-  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard')) or '$(TargetFramework)' == 'net461'">
+  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard')) or '$(TargetFramework)' == 'net461' or '$(TargetFramework)' == 'netcoreapp3.0'">
```

Comment:
This will potentially force an app using S.T.J built in .NET Core 3 to pull 5.0 from NuGet, right?
Could this cause other issues in the app? Not sure if I have an alternative to suggest though, like leaving the fix out sounds worse, and dropping support just because of this also sounds less optimal

Reply:
STJ 5.0.2. Has an explicit .NET Core 3.0 target, so I think it's OK.

Chances are high that folks that are still on 3.0 have this already through another dependency anyway.

Reply:
https://www.nuget.org/packages/System.Text.Json/5.0.2#supportedframeworks-body-tab

Reply:
I did a small test, and a .NET Core 3.0 console app will indeed work with STJ 5.0.2.  The next newest version (6.0.0) fails due to incompatible transitive dependencies, but 5.0.2 is fine.